lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc

```diff
@@ -0,0 +1,135 @@
+* Students analyze equivalence in pairs of Circles with absolute value and equivalence.
+* Students compute the result of expressions with absolute value and opposite.
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to about opposites and absolute value in combination.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* This activity involves a card sort. The easiest way to prepare for this is to give each pair of students an envelope containing the three sets of cards. Keep each set together with rubber bands or paperclips.
+
+|===
+
+
+== Translating with Absolute Value & Opposite @duration{25 minutes}
+
+=== Overview
+Students match Circles of Evaluation to expressions in words with absolute value and opposite, and then they consider how mathematical expressions correspond to those two representations.
+
+=== Launch
+
+Explain to students that they are going to receive @printable-exercise{three-sets-cards.adoc, three sets of cards}.
+
+The first set of cards includes Circles of Evaluation that use the `negate` and `num-abs` functions.
+
+The second set of cards translates those expressions into words.
+
+[.lesson-instruction]
+- With your partner, match each Circle card with the corresponding Words card.
+- Lay them out on the table in front of you so you can clearly see both the Circle and the Words.
+- Some Circles have more than one translation into words.
+- Discuss each match that you make.
+
+Circulate as students sort their cards, ensuring that they are carefully analyzing each card so that they see the connection between the Circle of Evaluation and its translation into words. Students will need to consider if translating the Circle into words involves reading from the outside-in or the inside-out.
+
+[.lesson-instruction]
+- Now, you’re going to receive a third set of cards - Mathematical Expressions.
+- One at a time, take turns analyzing each of these cards. After examining it closely, place it with the appropriate Circle / Words pairing.
+- Explain to your partner how and why you placed each card.
+- You and your partner must agree on each card’s placement before advancing to the next.
+
+Again, circulate during the activity. As students match expressions in words to their arithmetic expressions and Circles, invite them to use the Circle as a tool to help them translate to the other representations.
+
+=== Investigate
+
+[.lesson-instruction]
+- Each row on @printable-exercise{translating-abs-val-opp.adoc} represents a single arithmetic expression, written in three different forms. Sometimes, you will be provided with a Circle to translate, other times you will start with words or a mathematical expression.
+-  Fill in the empty spaces so that all three forms represent the same expression.
+- *MAYBE SHOULD ALSO ADD A WORDS-TO-CIRCLES MATCHING ACTIVITY LIKE IN TRANSLATING LESSON*
+
+=== Synthesize
+
+Here are some key take-aways from this lesson:
+
```

Comment:
Since circles of Evaluation don't exist outside of our curriculum and `num-abs` and `negate` are functions we defined it feels undesirable to have the big idea takeaways focus on those things rather than transferable math understanding... why would a teacher want to do these lessons if they can't see how they connect back to their learning goals?